src/FTPWindow.cpp

```diff
@@ -664,8 +657,6 @@ void TFTPWindow::DirlistChanged(BMessage *msg)
 	fRemoteFileView->Clear();
 
 	BString xpwd_result;
-	if (msg->FindString("xpwd", &xpwd_result) != B_OK) {
```

Comment:
`xpwd_result` is used below. Otherwise, you could have deleted its declaration as well.